jsapp/js/projects/myOrgProjectsRoute.tsx

```diff
@@ -0,0 +1,63 @@
+// Libraries
+import React, {useState, useEffect} from 'react';
+import {useNavigate} from 'react-router-dom';
+
+// Partial components
+import UniversalProjectsRoute from './universalProjectsRoute';
+import LoadingSpinner from 'js/components/common/loadingSpinner';
+
+// Stores, hooks and utilities
+import {useOrganizationQuery} from 'js/account/stripe.api';
+
+// Constants and types
+import {
+  ORG_VIEW,
+  HOME_ORDERABLE_FIELDS,
+  HOME_DEFAULT_VISIBLE_FIELDS,
+  HOME_EXCLUDED_FIELDS,
+} from './projectViews/constants';
+import {ROOT_URL} from 'js/constants';
+import {endpoints} from 'js/api.endpoints';
+import {PROJECTS_ROUTES} from 'js/router/routerConstants';
+
+/**
+ * Component responsible for rendering organization projects route
+ * (`#/organization/projects`).
+ */
+export default function MyOrgProjectsRoute() {
+  const orgQuery = useOrganizationQuery();
+  const navigate = useNavigate();
+  const [apiUrl, setApiUrl] = useState<string | null>(null);
+
+  // We need to load organization data to build the api url.
+  useEffect(() => {
+    if (orgQuery.data) {
+      setApiUrl(endpoints.ORG_ASSETS_URL.replace(':organization_id', orgQuery.data.id));
+    }
+  }, [orgQuery.data]);
+
+  // If user visits organization projects route and the organization is not MMO
+  // we redirect user out of here.
+  useEffect(() => {
```

Comment:
I would prefer if we handled this redirect logic at the route level. There is some redirect functionality in the `ValidateOrgPermissions` component I overhauled recently. What would you think about me updating that to include a prop for redirect route. Then you could remove this useEffect and use that wrapper component [here](https://github.com/kobotoolbox/kpi/pull/5240/files#diff-862f1d1178ac9c0f744ac0714f9d16de4e3f313754d88168cc82970cd9e99f2bR31)? Let me know what you think.

Reply:
Agree, let's do it at the route level :)

Reply:
Note to self: will fix this after #5262 gets merged